src/vs/gitpod/node/gitpodInsightsAppender.ts

```diff
@@ -191,7 +193,7 @@ function mapTelemetryData(eventName: string, data: any): RemoteTrackMessage | un
 			}
 			filterEvent.add(eventName);
 			return {
-				event: `vscode_${formatEventName(eventName)}`,
+				event: 'vscode_file_access',
```

Comment:
should I delete `fileGet` and `filePUT` as they are no reliable?

Reply:
It sounds like we can drop `fileGet`, but for `filePut` we don't have alternative, so let's keep it?

Reply:
removed `fileGet`